hyperjumptech/monika
# Fix issue 603 "Unknown property" error when using some alert configuration

# Monika Pull Request (PR)  

## What feature/issue does this PR fix  
1. Closing Issue #603 
2. Some "unknown" property error log when using alerts.

## How did you implement / how did you fix it  
1.  revert use of request.id back to request object

## How to test  
1. use this minial configuration, save it to monika.yml for instance
```yaml
probes:
  - id: 'test-1'
    name: test-1 probe
    description: status returns 200
    interval: 5
    requests:
      - url: https://httpbin.org/status/200
    alerts:
      - query: response.status != 200
        message: response status message

  - id: 'test-2'
    name: test-2 probe
    description: return 200
    interval: 5
    requests:
      - url: https://httpbin.org/status/200

notifications:
  - id: my-desktop-notif
    type: desktop
```
2. Run Monika with the setting `-c monika.yml -s 0 -r 1` to keep it at one repeats
3. Everything should run without any wierd logs. 

# Screenshot
Monika reporting events without wierd error on its log.
![image](https://user-images.githubusercontent.com/964106/154896644-4c8ba124-f685-4b0a-8e51-e03e856d64d2.png)

on Symon reporting properly: 
Running on symon mode, reporting its events properly
![image](https://user-images.githubusercontent.com/964106/155685770-532abec5-f631-499b-84bb-ace47f15c87e.png)
